app/controllers/DeparturesController.scala

```diff
@@ -38,4 +39,11 @@ class DeparturesController @Inject()(appConfig: AppConfig, cc: ControllerCompone
         Unauthorized
     }
   }
+
+  def getDepartures: Action[AnyContent] = Action {
```

Comment:
I'd just have this as the action get() rather than getDepartures() (we know we're getting departures, the controller name indicates this)